content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: 'Gateway API First Beta'
+date: 2022-07-12
+slug: gateway-api-first-beta
+---
+
+**Author:** Shane Utt (Kong)
+
+We're pleased to announce that key Gateway API resources are graduating to
+beta with the `v0.5.0` release of the API. In addition to `GatewayClass`,
+`Gateway` and `HTTPRoute` graduating to beta, this release introduces several
+new concepts to the API that we'll cover below.
+
+## What is Gateway API
+
+Gateway API is a collection of resources centered around `Gateway` resources
```

Comment:
I'd make Gateway be a hyperlink to https://gateway-api.sigs.k8s.io/beta1/api-types/gateway/

Reply:
@sftim the link you provided somehow runs into a 404 for me. I'm assuming you mean this link - https://gateway-api.sigs.k8s.io/api-types/gateway/ or is there something that is supposed to be there but isn't?

Reply:
Hmm. yes. I thought I saw a link to https://gateway-api.sigs.k8s.io/v1alpha1/api-types/gateway/ but that doesn't seem to work when I check now.

Linking to https://gateway-api.sigs.k8s.io/api-types/gateway/ seems right.

Reply:
```suggestion
Gateway API is a collection of resources centered around [Gateway](https://gateway-api.sigs.k8s.io/api-types/gateway/) resources
```